VirtoCommerce.CatalogModule.Data/Services/CategoryServiceImpl.cs

```diff
@@ -100,6 +100,11 @@ public virtual void Delete(string[] categoryIds)
                 .Select(c => new GenericChangedEntry<Category>(c, EntryState.Deleted))
                 .ToList();
 
+            foreach (var category in categories)
```

Comment:
Here i think we need to add method DeleteSeoForObject**s** for array of objects to ICommerceService, and make current DeleteSeoForObject obsolete. It would allow us to avoid multiple requests